src/dex-ui/pages/dao/CreateADAO/forms/CreateTokenNFTDAOForm.tsx

```diff
@@ -0,0 +1,242 @@
+  const isErrorDialogOpen = isCreateNFTFailed || isGetAccountDetailsFailed || isTransactionDetailsFailed;
+
+  function getLoadingDialogMessage(): string {
+    if (isLoadingDialogOpen)
+      return `Please confirm the create ${governance.newNFT.name} 
+    NFT transaction in your wallet to proceed.`;
+    return "";
+  }
+
+  function getErrorDialogMessage(): string {
+    if (isGetAccountDetailsFailed) return getAccountDetailsError?.message;
+    if (isCreateNFTFailed) return createNFTError?.message;
+    if (isTransactionDetailsFailed) return transactionDetailsError?.message;
+    return "";
+  }
+
+  const loadingDialogMessage = getLoadingDialogMessage();
+  const errorDialogMessage = getErrorDialogMessage();
+
+  useEffect(() => {
+    if (!isNil(transactionDetails)) {
+      setValue("governance.newNFT.id", transactionDetails[0].entity_id, { shouldValidate: true });
+      setValue("governance.newNFT.treasuryWalletAccountId", governance.newNFT.tokenWalletAddress, {
+        shouldValidate: true,
+      });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [transactionDetails]);
+
+  function handleCreateNFTSuccess(transactionResponse: TransactionResponse) {
+    const message = `${governance?.newNFT?.symbol} NFT was successfully created`;
+    handleTransactionSuccess(transactionResponse, message);
+  }
+
+  async function createNewToken() {
+    const isCreateTokenDataValid = await trigger([
+      "governance.newNFT.name",
+      "governance.newNFT.symbol",
+      "governance.newNFT.maxSupply",
+      "governance.newNFT.tokenWalletAddress",
+    ]);
+    if (isCreateTokenDataValid) {
+      const nftDAOFormData = formValues as CreateANFTDAOForm;
+      const {
+        governance: {
+          newNFT: { name, symbol, maxSupply, tokenWalletAddress },
+        },
+      } = nftDAOFormData;
+
+      const {
+        key: { key },
+      } = await getAccountDetails(undefined);
+
+      createNFT({
+        name,
+        symbol,
+        maxSupply,
+        supplyKey: key,
+        tokenWalletAddress,
+      });
+    }
+  }
+  return (
+    <>
+      <Flex gap="1.5rem" direction="column" width="100%">
+        <DAOFormContainer>
+          <Flex direction="row" gap="4">
+            <FormInput<"governance.newNFT.name">
+              flex="3"
+              inputProps={{
+                id: "governance.newNFT.name",
+                flex: 3,
+                label: "NFT Name",
+                type: "text",
+                placeholder: "Enter Name",
+                register: {
+                  ...register("governance.newNFT.name", {
+                    required: { value: true, message: "NFT name is required." },
+                  }),
+                },
+              }}
+              isInvalid={Boolean(errors.governance?.newNFT?.name)}
+              errorMessage={errors.governance?.newNFT?.name && errors.governance?.newNFT?.name.message}
+            />
+            <Center height="64px">
```

Comment:
we can use "rem" instead of "px"

Reply:
you are right. made the suggested changes.